librz/core/cmd_descs/cmd_analysis.yaml

```diff
@@ -1453,3 +1453,49 @@ commands:
               - name: class_name
                 type: RZ_CMD_ARG_TYPE_STRING
                 optional: true
+  - name: as
+    summary: Syscalls
+    subcommands:
+      - name: as
+        summary: Show syscall and arguments
+        cname: analysis_syscall_show
+        args:
+          - name: syscall
+            type: RZ_CMD_ARG_TYPE_NUM
+            optional: true
+      - name: asl
+        summary: List syscalls
+        cname: analysis_syscall_print
+        type: RZ_CMD_DESC_TYPE_ARGV_STATE
+        default_mode: RZ_OUTPUT_MODE_STANDARD
+        modes:
+          - RZ_OUTPUT_MODE_STANDARD
+          - RZ_OUTPUT_MODE_JSON
+        args: []
```

Comment:
In the past it was possible to list just one syscall. I think we can keep this feature or actually we could even accept an array of strings and return all syscalls matching the array.

Reply:
I moved this feature into the `asn` and `asr` commands respectively.